src/classpath/assets/features/classpathConfiguration/components/ProjectSelector.tsx

```diff
@@ -63,7 +66,7 @@ const ProjectSelector = (): JSX.Element | null => {
         {(projectType === ProjectType.Gradle || projectType === ProjectType.Maven) &&
           <div className="mt-1">
             <span className="warning">
-              Below settings are only applicable for non-build tool projects. For the {projectType} project, please edit them in the <a href="" onClick={() => handleOpenBuildFile()}>{buildFile}</a> file.
+              Below settings are only editable for projects without build tools. For {projectType} project, please edit the <a href={encodeCommandUriWithTelemetry(WEBVIEW_ID, "classpath.openBuildToolDoc", "java.helper.openUrl", [buildFileDocUrl])}>entries</a> in <a href="" onClick={() => handleOpenBuildFile()}>{buildFile}</a>.
```

Comment:
Here about telemetry you don't know whether user clicks doc for Maven or Gradle. 

One suggestion is to copy paste, using almost identical DOM nodes for Gradle and Maven, seperately. One advantage is, we can further customize the message (they don't have to be the same).